user_docs/en/userGuide.md

```diff
@@ -323,33 +323,33 @@ To install an add-on:
 
 1. With the focus on an add-on you would like to install, press `enter`.
 1. The actions menu opens with a list of actions; the first action is "Install".
-1. To install the add-on, press `i` or `downArrow` to "install" and press `enter`.
+1. To install the add-on, press `i` or `downArrow` to "Install" and press `enter`.
 1. The focus returns to the add-on in the list and NVDA will read the details about the add-on.
 1. The "Status" information reported by NVDA changes from "Available" to "Downloading".
 1. Once the add-on has finished downloading, it will change to "Downloaded. Pending install".
 1. Repeat with any other add-ons you would like to install at the same time.
 1. Once finished, press `tab` until the focus is on the "Close" button, then press `enter`.
 1. The downloaded add-ons will start the installation process once the Add-on Store is closed.
-Add-ons may create dialogs during the installation process that you will need to respond to.
+During the installation process, add-ons may display dialogs that you will need to respond to.
 1. When the add-ons have been installed, a dialog appears advising that changes were made, and you must restart NVDA for the add-on installation to complete.
 1. Press `enter` to restart NVDA.
 
 #### Managing installed add-ons {#ManagingInstalledAddons}
-Press `control+tab` to move between the tabs or sections of the Add-on Store.
+Press `control+tab` to move between the tabs of the Add-on Store.

```

Comment:
What are the sections of the Add-on Store? If it's a synonym of "tab", it's not needed IMO.